src-tool/Pact/Analyze/Check.hs

```diff
@@ -403,7 +411,7 @@ verifyFunctionInvariants (CheckEnv tables _consts _pDefs moduleData caps gov)
       [] -> pure $ invsMap & traverse .~ []
 
       _ -> ExceptT $ catchingExceptions $ runSymbolic $ runExceptT $ do
-        lift $ SBV.setTimeOut 1000 -- one second
+        lift $ SBV.setTimeOut timeout
```

Comment:
That was an important comment that should be preserved here or in the comment above for the `timeout` value.